docs/extensions.html

```diff
@@ -109,23 +89,30 @@ <h1 class="title">
 
 <section class="section">
   <div class="container">
-    {% for extension in page.extensions %}
-      <a class="box" href="{{ extension.url }}">
-        <div class="columns">
-          <div class="column is-one-third">
-            <h3 class="title is-4">
-              <strong>{{ extension.name }}</strong>
-            </h3>
+    <div class="columns is-multiline">
+      {% for extension in page.extensions %}
+      <div class="column is-3">
+        <div class="card">
+          <div class="card-image">
+            <figure class="image is-16by9 is-contained">
+              <img src="{{site.url}}/images/extensions/{{ extension.slug }}.png">
+            </figure>
+          </div>
+          <div class="card-content">
+            <p class="title is-5">{{ extension.name }}</p>
             <h4 class="subtitle is-6">
-              {{ extension.url | remove:'http://' | remove:'https://' | remove:'www.' }}
+            {{ extension.description }}
             </h4>
           </div>
-          <div class="column">
-            <img src="{{site.url}}/images/extensions/{{ extension.slug }}.png" width="{{ extension.width }}" height="{{ extension.height }}">
+          <div class="card-footer">
+            <div class="card-footer-item">
+              <a href="{{ extension.url }}"><i class="fa fa-eye"></i> Extension page</a>
```

Comment:
This makes the click zone way smaller than before. I don't see it as an improvement.